mala/common/physical_data.py

```diff
@@ -277,15 +276,16 @@ def write_to_openpmd_iteration(self, iteration, array,
         atoms_ase = self._get_atoms()
         if atoms_ase is not None:
             atoms_openpmd = iteration.particles["atoms"]
+            # @todo: metadata
             atomic_positions = atoms_ase.get_positions()
             positions = io.Dataset(atomic_positions[0].dtype,
                                    atomic_positions[0].shape)
 
-            # @Franz: In the example online this is done separately for
-            # x, y, z, but isn't it better to do that indvidually for atoms?
```

Comment:
Both approaches are probably valid here, yours maybe a little bit more memory-efficient, but I don't think it'll make a difference.
I've added the required `positionOffset` field, but no metadata yet.